pkg/storage/pod_test.go

```diff
@@ -137,8 +146,18 @@ var _ = Describe("Pod storage", func() {
 		By("storing second batch with exactly same metric")
 		s.Store(batch)
 
-		By("return empty results for pod1")
-		checkPodResponseEmpty(s, podRef)
+		By("return correct results for pod1")
```

Comment:
ditto

Reply:
this test should not be impacted.

Reply:
This test is about repeated metric point, I don't expect we should change behavior here.